### PR TITLE
Add License Metadata

### PR DESCRIPTION
**Pull request by ciaranRoche**

### CODE REVIEW

File: licenses/licenses.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0'?>
+<licenseSummary>
+    <project>saml-android-app</project>
+    <version>0.0.0</version>
```
Comment (ciaranRoche) [inline licenses/licenses.xml:4]: @lgriffin same as the other android PR, can't find a version number in the repo

Comment (lgriffin) [reply licenses/licenses.xml:4]: Version number isn't hugely important as that metadata is not really used anywhere meaningful. I'd leave it at 0.0.0 and we can PR against it if we do come up with a better number.

**Comment by lgriffin** [comment]
@ciaranRoche let me know when I can merge this

File: licenses/licenses.xml (hunk 1 above)
Comment (ciaranRoche) [reply licenses/licenses.xml:4]: Awesome, just wanted to double check, feel free to merge if you are happy

**Pull request merged**